### PR TITLE
Update installation files and instructions

### LINKED ISSUE #117: Installation check fails on notebook
Reported by @alaity47 

`python check_env.py` fails to find module "notebook". 

```
(navo-env) garibaldi:navo-workshop laity$ python check_env.py 
Found jupyter 
Error: Failed import: No module named 'notebook'
Found numpy 1.24.3
Found matplotlib 3.7.1
Found jupyterlab 4.0.0
Found astropy 5.2.2
Found pyvo 1.4.1
Found astroquery 0.4.6

There are errors that you must resolve before running the tutorials.
```

Also, we don't check for the existence of the "scipy" module even though it is installed in the notebook.

Notes:
* Newest Intel miniconda threw an "incompatible architecture" error. Second-newest miniconda worked.
* Step 5 of the [installation instructions](https://nasa-navo.github.io/navo-workshop/00_SETUP.html) took a long time to run some of its pieces, might want to to note it's expected to spin for a while.

### PR DESCRIPTION
**Pull request by stargaser**
* Remove 'notebook' from `check_env.py` as it is not installed with Jupyterlab 4.0.
* Add a check for 'scipy' and a minimum version of 1.0
* Add Mambaforge as an alternative to Miniconda
* Note that the environment installation can take a while

Fixes #117

### CODE REVIEW

**Review by bsipocz — APPROVED** [review]
All looks good to me, though I may suggest dropping conda altogether. Having both conda and mamba could be confusing when someone is totally new to the ecosystem.

**Review by bsipocz — COMMENTED** [review]
I was just pointed out that changing the solver for conda with e.g. `--solver=libmamba` could work out of the box. For this `conda-libmamba-solver` should be installed. 

Maybe this more straightforward as it doesn't require any introduction or explanation of what mamba is?

File: 00_SETUP.md [hunk 1]
```diff
@@ -16,7 +16,15 @@ conda info
...
 
 If Miniconda is not already installed, follow these instructions for your
-operating system: https://docs.conda.io/en/latest/miniconda.html
+operating system: [https://docs.conda.io/en/latest/miniconda.html](https://docs.conda.io/en/latest/miniconda.html)
+
+(If you have trouble installing Miniconda, an alternative is to install
+Mambaforge. You can obtain an installer for your operating system at
+[https://github.com/conda-forge/miniforge#mambaforge](https://github.com/conda-forge/miniforge#mambaforge).
+Then follow the installation instructions at 
+[https://github.com/conda-forge/miniforge#mambaforge](https://github.com/conda-forge/miniforge#mambaforge).
```
Comment (tomdonaldson) [inline 00_SETUP.md:25]: Minor URL tweak:
```suggestion
[https://github.com/conda-forge/miniforge#install](https://github.com/conda-forge/miniforge#install).
```

**Review by tomdonaldson — COMMENTED** [review]
I think this has a nice balance.  I was not familiar with mambaforge, so am hesitant to leap to that just before the workshop.  I will play with it to become more familiar so that I can help with installation issues and try to answer some of my own questions.

Just updating my local miniconda didn't give me the "incompatible architecture" problem.  I'm assuming that issue was on an Intel Mac.  Was it with just the latest Python 3.10 version or  was it also seen with a fresh install of the latest Python 3.9 version?

**Comment by stargaser** [comment]
> Just updating my local miniconda didn't give me the "incompatible architecture" problem. I'm assuming that issue was on an Intel Mac. Was it with just the latest Python 3.10 version or was it also seen with a fresh install of the latest Python 3.9 version?

Anastasia encountered this on her Intel Mac with the latest version which must be Python 3.10. The second newest worked which must be the Python 3.9 version.

**Comment by bsipocz** [comment]
I would still suggest trying adding the different solver, that shouldn't add confusion, but would provide a much faster experience.

**Comment by tomdonaldson** [comment]
> I would still suggest trying adding the different solver, that shouldn't add confusion, but would provide a much faster experience.

I won't object to that, though I think it obligates some of us to do a little testing of the new instructions.  Should we do that here, or should I merge this and look for a proposed change in another PR?

**Comment by tomdonaldson** [comment]
FWIW, I haven't been able to reproduce the miniconda error.  Obviously there could be a lot of variables, but both of these downloaded (with correct SHA256 hash), installed and worked on my Intel Mac (MacOS 12.6.6 Monterey):
```
Miniconda3-py310_23.3.1-0-MacOSX-x86_64.pkg
Miniconda3-py310_23.3.1-0-MacOSX-x86_64.sh
```
This does make me wonder if the problem might not be very widespread.

**Comment by stargaser** [comment]
Okay, I've updated to remove mamba and add in the libmamba solver. See if this makes sense.

**Comment by stargaser** [comment]
Oops, never mind, it's been merged, I missed that. Let's pu the change in another PR.

**Pull request merged**